osmaxx/excerptexport/models/excerpt.py

```diff
@@ -84,6 +85,13 @@ def color(self):
     def extent(self):
         return self.bounding_geometry.extent
 
+    @property
+    def has_running_exports(self):
+        return any([not i.can_be_deleted for es in self.extraction_orders.all() for i in es.exports.all()])
```

Comment:
`i` will be corrected.

generator expression is a good idea, I really should use them more often.

Makes sense, the expression got a bit out of hands, when using the `can_be_deleted` from exports, I'm not fan of this implementation in general. Do you have any smart suggestions (besides renaming)?